samples/peripheral/radio_test/conf/remote_shell/usb.dts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		zephyr,shell-ipc = &ipc0;
+		ncs,remote-shell-uart = &cdc_acm0;
+	};
+};
+
+&zephyr_udc0 {
+	cdc_acm0: cdc_acm0 {
+		compatible = "zephyr,cdc-acm-uart";
+		label = "CDC_ACM_0";
```

Comment:
No label properties, please fix it everywhere.

Reply:
Sure, removed